Fix CI test failure: Replace non-existent 'health check' command with 'version' command

## Problem

The CI/CD pipeline in `.github/workflows/test-installation.yml` is failing because it attempts to run a non-existent `oboyu health check` command.

**Error:**
```
Run test-env/bin/oboyu health check
Usage: oboyu [OPTIONS] COMMAND [ARGS]...
Try 'oboyu --help' for help.
Error: No such command 'health'.
Error: Process completed with exit code 2.
```

## Root Cause

The `health` command is not implemented in the CLI interface. The current available commands are:
- `index` - Index documents for search
- `search` - Search documents with GraphRAG enhancement  
- `build-kg` - Build knowledge graph from indexed documents
- `deduplicate` - Deduplicate entities in knowledge graph
- `mcp` - Run MCP server for AI assistant integration
- `clear` - Clear command
- `status` - Status command
- `version` - Display version information

While health checking functionality exists internally in the codebase, it's not exposed as a CLI command.

## Solution

Replace the failing `health check` command with the `version` command for basic functionality testing.

**File to modify:** `.github/workflows/test-installation.yml`

**Change required:**
```yaml
# Before:
- name: Run basic functionality test
  run: |
    test-env/bin/oboyu health check

# After:
- name: Run basic functionality test
  run: |
    test-env/bin/oboyu version
```

## Benefits of using `version` command

1. **Simple and reliable** - No external dependencies required
2. **Installation verification** - Confirms CLI is properly installed and functional
3. **Lightweight** - No database files or configuration needed
4. **Clear output** - Returns version information for verification

This change will resolve the CI failure and provide appropriate basic functionality testing for the installation workflow.

--- a/.github/workflows/test-installation.yml
+++ b/.github/workflows/test-installation.yml
@@ -65,7 +65,7 @@ jobs:
           
       - name: Run basic functionality test
         run: |
-          test-env/bin/oboyu health check
+          test-env/bin/oboyu version
           
   test-docker-install:
     runs-on: ubuntu-latest

